modin/pandas/series.py

```diff
@@ -516,7 +523,10 @@ def f(x):
                     return f(self)
                 query_compiler = self.map(f)._query_compiler
         if return_type not in ["DataFrame", "Series"]:
-            return query_compiler.to_pandas().squeeze()
+            try:
+                return query_compiler.to_pandas().squeeze()
```

Comment:
When does this throw an `Exception`?

Reply:
It can throws an `Exception` in the case if result of applying of function is not a `query_compiler`, but scalar (such result can occurs for `sum` or `count` functions, for example). This expression was reworked to explicitly check result type and corresponding comment is added.